linera-service/src/cli_wrappers/remote_net.rs

```diff
@@ -20,14 +20,19 @@ pub struct RemoteNetTestingConfig {
 }
 
 impl RemoteNetTestingConfig {
+    /// Creates a new [`RemoteNetTestingConfig`] for running tests with an external Linera
+    /// network.
+    ///
+    /// The `faucet_url` is used to connect to the network and obtain its configuration,
+    /// as well as to create microchains used for testing. If the parameter is [`None`],
```

Comment:
Will that link to https://doc.rust-lang.org/std/option/enum.Option.html#variant.None? Not sure if that's useful.